src/qt/pivx/governancewidget.cpp

```diff
@@ -142,7 +142,23 @@ void GovernanceWidget::onVoteForPropClicked(const ProposalInfo& proposalInfo)
 
 void GovernanceWidget::onCreatePropClicked()
 {
-    if (!walletModel || !governanceModel) return;
+    if (!walletModel || !governanceModel || !clientModel) return;
+
+    if (!governanceModel->isTierTwoSync()) {
+        inform(tr("Please wait until the node is fully synced"));
+        return;
+    }
+
+    // Do not allow proposals submission 1440 blocks away (1 day) from the next superblock
+    // The budget finalization could have been submitted and the user would never know it, losing the first superblock.
+    // future: customizable future superblock height selection (for now, we are automatically using the next superblock).
+    const int chainHeight = clientModel->getLastBlockProcessedHeight();
+    const int nextSuperblock = governanceModel->getNextSuperblockHeight();
+    const int acceptedRange = walletModel->isTestNetwork() ? 10 : 1440;
+    if (nextSuperblock - acceptedRange < chainHeight) {
+        inform(tr("Cannot create proposal, superblock is too close. Need to wait %d blocks").arg(nextSuperblock - chainHeight));
```

Comment:
Arguments in `tr()` strings need to be numbered for value substitution to happen
`%d` -> `%1`
```diff
- inform(tr("Cannot create proposal, superblock is too close. Need to wait %d blocks").arg(nextSuperblock - chainHeight));
+ inform(tr("Cannot create proposal, superblock is too close. Need to wait %1 blocks").arg(nextSuperblock - chainHeight));
```

Reply:
upps, updated. My head is focused in 2659..